File: samples/csharp_dotnetcore/17.multilingual-bot/Translation/Model/TranslatorResult.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using Newtonsoft.Json;

namespace Microsoft.BotBuilderSamples.Translation.Model
{
    /// <summary>
    /// Translation result from Translator API v3.
    /// </summary>
    internal class TranslatorResult
    {
        [JsonProperty("text")]
        public string Text { get; set; }

        [JsonProperty("to")]
        public string To { get; set; }
    }
}

File: archive/samples/java_springboot/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/translation/model/TranslatorResult.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.microsoft.bot.sample.multilingual.translation.model;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Translation result from Translator API v3.
 */
public class TranslatorResult {
    @JsonProperty("text")
    private String text;

    @JsonProperty("to")
    private String to;

    /**
     * Gets the translation result text.
     * @return Translation result.
     */
    public String getText() {
        return this.text;
    }

    /**
     * Sets the translation result text.
     * @param withText Translation result.
     */
    public void setText(String withText) {
        this.text = withText;
    }

    /**
     * Gets the target language locale.
     * @return Locale.
     */
    public String getTo() {
        return this.to;
    }

    /**
     * Sets the target language locale.
     * @param withTo Target locale.
     */
    public void setTo(String withTo) {
        this.to = withTo;
    }
}
